addons/website_event/views/snippets/s_events.xml

```diff
@@ -83,12 +83,13 @@
         <we-many2many t-if="snippet_name == 'event_upcoming_snippet'"
                       string="Event Tags"
                       data-name="event_tag_opt"
-                      data-model="event.event"
-                      data-m2o-field="tag_ids"
+                      data-model="event.tag"
+                      data-fakem2m="true"
                       data-domain='[["category_id.website_published", "=", true], ["color", "not in", ["0", false]]]'
                       data-limit="10"
-                      data-set-tags=""
                       data-attribute-name="filterByTagIds"
+                      data-fields="[&quot;category_id&quot;]"
```

Comment:
Looks a little strange but I'm guessing it's what the JS expects ^^

Reply:
Yes, it seems so :astonished: 
I tried with quotes but it didn't work...
I found another occurence here : https://github.com/odoo/odoo/blob/16.0/addons/website/views/snippets/snippets.xml#L1262